src/views/EmergencyThreeWDetails/index.tsx

```diff
@@ -0,0 +1,265 @@
+                                                value={activity?.household_count}
+                                                strongLabel
+                                            />
+                                        )}
+                                    </>
+                                ) : (
+                                    <div className={styles.disaggregated}>
+                                        {/* TODO */}
+                                    </div>
+                                )}
+                                {activity?.supplies && Object.keys(activity.supplies).length > 0
+                                && (
+                                    <Container
+                                        heading={strings.emergencySupplies}
+                                        headingLevel={3}
+                                        headerClassName={styles.supplyHeader}
+                                        childrenContainerClassName={styles.supplyContent}
+                                    >
+                                        {(Object.keys(
+                                            activity?.supplies,
+                                        ) as unknown as number[]).map((k) => (
+                                            <TextOutput
+                                                key={k}
+                                                label={supplyIdToTitleMap?.[k]}
+                                                value={activity?.supplies?.[k]}
+                                                strongLabel
+                                            />
+                                        ))}
+                                    </Container>
+                                )}
+                                {Object.keys(activity?.custom_supplies).length > 0
+                                && (
+                                    <Container
+                                        heading={strings.emergencyCustomSupplies}
+                                        headingLevel={3}
+                                        headerClassName={styles.customSupplyHeader}
+                                        childrenContainerClassName={styles.customSupplyContent}
+                                    >
+                                        {(Object.keys(
+                                            activity?.custom_supplies,
+                                        ) as unknown as number[]).map((k) => (
+                                            <TextOutput
+                                                key={k}
+                                                label={k}
+                                                value={activity?.custom_supplies?.[k]}
+                                                strongLabel
+                                            />
+                                        ))}
```

Comment:
Use Object.entries to iterate over an object type. We don't need to get the values by accessing again.